src/js/components/ToggleButtonGroup/stories/controlledGroup.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
```

Comment:
For the storybook examples, can we:
- Put these under "Controls" section as opposed to "Inputs"? 
- Capitalize the file names to be consistent with other stories?
- For story naming, maybe "Controlled", "Multiple", "Simple", "Icons"